content/first-steps.md

```diff
@@ -112,3 +112,44 @@ $ npm run start:dev
 ```
 
 This command will watch your files, automatically recompiling and reloading the server.
+
+#### Linting and formatting
+
+[Nest CLI](/cli/overview) provides best effort to scaffold a reliable development workflow at scale. Thus, a generated Nest project comes with both a code **linter** and **formatter** preinstalled (respectively [eslint](https://eslint.org/) and [prettier](https://prettier.io/)).
```

Comment:
```suggestion
[Nest CLI](/cli/overview) provides best effort to scaffold a reliable development workflow at scale. Thus, a generated Nest project comes with both a code **linter** and **formatter** preinstalled (respectively [ESLint](https://eslint.org/) and [Prettier](https://prettier.io/)). Do note that you can use any other tool for such tasks, and they are not maintained by the NestJS core team.
```

I guess we should mention somehow that ESLint & Prettier are not related with NestJS directly, otherwise beginners on Node might think that they are.